packages/components/src/theme/query-model.scss

```diff
@@ -97,49 +96,49 @@
         line-height: 16px;
         margin-right: 8px;
         padding: 8px 8px 8px 32px;
-        color: $omnibox-item-color;
-        background-color: $omnibox-item-bg;
+        color: $gridaction-item-color;
```

Comment:
Why no hyphen for `gridaction`? (I keep reading it as `gradation` for some reason.)

Reply:
no reason, updated.